### PR TITLE
[TR] PIM-4675: Fix translation loading when no hash nav

### PR DESCRIPTION
**Pull request by grena**

### CODE REVIEW

File: src/Oro/Bundle/TranslationBundle/Resources/views/requirejs.config.js.twig [hunk 1]
```diff
@@ -1,21 +1,8 @@
 require({
-    shim: {
-        'oro/translations': {
-            deps: ['oro/translator', 'translator'],
-            init: function(__) {
-                return __;
-            }
+    config: {
+        'oro/translator': {
+            {#messagesUrl: '{{ asset('js/translation/%s.js'|format(locale)) }}'#}
```
Comment (juliensnz) [inline src/Oro/Bundle/TranslationBundle/Resources/views/requirejs.config.js.twig]: clean comments

File: src/Oro/Bundle/TranslationBundle/Resources/views/requirejs.config.js.twig [hunk 2]
```diff
@@ -1,21 +1,8 @@
...
-            'oro/translator': 'oro/translations'
-        },
-        'oro/translations': {
-            'oro/translator': 'oro/translator'
-        }
-    },
-    paths: {
-        'oro/translations': '{{ url('oro_translation_jstranslation')[0:-3] }}'
     }
 });
```
Comment (juliensnz) [inline src/Oro/Bundle/TranslationBundle/Resources/views/requirejs.config.js.twig]: add a blank line

**Comment by juliensnz** [comment]
GTM

**Comment by solivier** [comment]
GTM

**Comment by jmleroux** [comment]
so much simpler.

GTM :+1:

**Pull request merged**